sdk/include/opentelemetry/sdk/metrics/aggregator/aggregator.h

```diff
@@ -45,7 +45,6 @@ class Aggregator
    * @return none
    */
   virtual void update(T val) = 0;
-
```

Comment:
Are these intentional?

Reply:
This happened when running the format tool, it's not intentional. I can remove